aclocal.m4

```diff
@@ -1040,7 +1040,11 @@ AC_DEFUN([EGG_TCL_TCLCONFIG],
     if test -r ${TCL_BIN_DIR}/tclConfig.sh; then
       . ${TCL_BIN_DIR}/tclConfig.sh
       # OpenBSD uses -pthread, but tclConfig.sh provides that flag in EXTRA_CFLAGS
-      TCL_PTHREAD_LDFLAG=`echo $TCL_EXTRA_CFLAGS | grep -o -- '-pthread'`
+      if [[ "$(echo $TCL_EXTRA_CFLAGS | grep -- "-pthread")" != "" ]]; then
```

Comment:
Don't use the [[ bashism but `test`. Would prefer for conformity:
```
TCL_PTHREAD_LDFLAG=`echo $TCL_EXTRA_CFLAGS | grep -- '-pthread'`
if test "x$TCL_PTHREAD_LDFLAG" != x; then
  TCL_PTHREAD_LDFLAG="-pthread"
fi
```
Same for below.

Reply:
fixed, tested again, ok.